addons/website_slides/tests/test_slide_slide.py

```diff
@@ -22,6 +22,43 @@ def test_slide_create_vote_constraint(self):
                 'vote': 2,
             })
 
+    def test_compute_category_completion_time(self):
```

Comment:
Note that tests should be ordered alphabetically in a given class. Reason is that they are executed alphabetically, hence it is easier to debug (find / update tests) when their execution order matches their definition order.

Reply:
Thank you so much for your time 🙏🏽 hope it's good now :)